resources/scripts/ttx_diff.py

```diff
@@ -163,22 +163,6 @@ def find_table(ttx, tag):
     return el[0]
 
 
-def sort_name(ttx):
-    name = find_table(ttx, "name")
-    records = sorted(
-        list(name),
-        key=lambda el: (
-            int(el.attrib["nameID"]),
-            el.attrib["platformID"],
-            el.attrib["platEncID"],
-            el.attrib["langID"],
-        ),
-    )
-    name.clear()
-    for record in records:
-        name.append(record)
-
-
```

Comment:
is it possible that this was the only change required for this patch to work, and that the changes above are unnecessary?

Reply:
It is! https://github.com/googlefonts/fontc/pull/326.